src/stm32/Makefile

```diff
@@ -70,3 +70,16 @@ deployer-y += generic/armcm_boot.c generic/armcm_reset.c $(mcu-y)
 CFLAGS_deployer.elf += -nostdlib -lgcc -lc_nano
 CFLAGS_deployer.elf += -T $(OUT)src/generic/armcm_deployer.ld
 $(OUT)deployer.elf: $(OUT)src/generic/armcm_deployer.ld
+
+# Flash rules
+lib/hidflash/hid-flash:
+	@echo "  Building hid-flash"
+	$(Q)make -C lib/hidflash
+
+flash: $(OUT)katapult.bin lib/hidflash/hid-flash
+	@echo "  Flashing $< to $(FLASH_DEVICE)"
+	$(Q)$(PYTHON) ./scripts/flash_usb.py -t $(CONFIG_MCU) -d "$(FLASH_DEVICE)" -s "$(CONFIG_FLASH_APPLICATION_ADDRESS)" $(if $(NOSUDO),--no-sudo) $(OUT)katapult.bin
+
```

Comment:
This doesn't look correct to me.  We would not want to flash Katapult at the `application address`, it should always be at the start address.